.github/workflows/maven.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        java-version: [ 8 ]
+        java-version: [ 8, 11, 17 ]
```

Comment:
Why not add everything from 8 all the way to 17?

Reply:
I am not sure I follow this "termurin" explanation. Are you saying that it is impossible to do similar as we do in STARTS [here](https://github.com/TestingResearchIllinois/starts/blob/master/.github/workflows/maven.yml#L19)?

Reply:
Is there some official documentation on when one should use v1 vs. v3?

Reply:
I don't know. Those probably use some different way to locate jars. Can you remove 12, 13, and 14 from CI, and open an issue with this error message so that we can come back to fix them later?

Reply:
sounds good